### ISSUE TITLE
Better signup and login pages

### ISSUE THREAD

**Issue by callmeVAMP** [govind-kumarr/simple_rag_dashboard#15]
Here ui can be improved for signup and login pages. Assign me this under gssoc 2024

**Comment by govind-kumarr** [comment]
Hi, @callmeVAMP  Thanks for showing interest in this project. Yeah, i know the UI is not that much attractive because i was more focused on functionality. You can improve but please make sure you go through the CSS guidelines mentioned in the chatbot_rjs directory.

**Comment by RakshitaJain2126** [comment]
Can you assign it to me as well? I also want to work on it.

**Comment by palayushi293** [comment]
pls assign this task to me i have a better knowledge of UI/Ux designs
Waiting for your response

**Comment by govind-kumarr** [comment]
Hi, @palayushi293  @RakshitaJain2126  Thanks for showing interest in this project.
Can you design better designs for the below pages? I an create issues fro all of them.

![Screenshot (632)](https://github.com/govind-kumarr/simple_rag_dashboard/assets/106414173/a8ab5119-8a4c-40f7-97ee-5c9da2bcd002)
![Screenshot (633)](https://github.com/govind-kumarr/simple_rag_dashboard/assets/106414173/5ec7d362-2fae-4ecb-bc78-7783a9d56dd4)
![Screenshot (634)](https://github.com/govind-kumarr/simple_rag_dashboard/assets/106414173/d84d4c59-6825-47ad-a486-2c026aa8bfb7)
![Screenshot (635)](https://github.com/govind-kumarr/simple_rag_dashboard/assets/106414173/868cdac4-0b09-4418-a650-7865cdd9c44a)

**Comment by govind-kumarr** [comment]
You can setup project locally to see it better.

**Comment by palayushi293** [comment]
yes sir,,

**Comment by govind-kumarr** [comment]
@palayushi293  great create issues for the pages you want to design i will be happy to assign those to you.

**Comment by RakshitaJain2126** [comment]
> Hi, @palayushi293 @RakshitaJain2126 Thanks for showing interest in this project. Can you design better designs for the below pages? I an create issues fro all of them.
> 
> ![Screenshot (632)](https://private-user-images.githubusercontent.com/106414173/329840564-a8ab5119-8a4c-40f7-97ee-5c9da2bcd002.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1MTU2MTcsIm5iZiI6MTcxNTUxNTMxNywicGF0aCI6Ii8xMDY0MTQxNzMvMzI5ODQwNTY0LWE4YWI1MTE5LThhNGMtNDBmNy05N2VlLTVjOWRhMmJjZDAwMi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxMlQxMjAxNTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04NTA2OWExZDhkMjlkMmRiODRhMmNlNzRkY2QyZTc5MDE1MDQxMGM4ODFkY2QxMmJhOGFmYTk1MzM4Y2VhZTBlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.3i09Nmsap9oz_rgbiJkyvAKr7DgtTO87Cwgz-ZzkkZY) ![Screenshot (633)](https://private-user-images.githubusercontent.com/106414173/329840568-5ec7d362-2fae-4ecb-bc78-7783a9d56dd4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1MTU2MTcsIm5iZiI6MTcxNTUxNTMxNywicGF0aCI6Ii8xMDY0MTQxNzMvMzI5ODQwNTY4LTVlYzdkMzYyLTJmYWUtNGVjYi1iYzc4LTc3ODNhOWQ1NmRkNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxMlQxMjAxNTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mOWI5ODZiMzlkNTk1MTQxMWM4NDczM2M5ZWIxNzRiMzM1MDdiOTU5NmIwYzg2YWYwOTBmYzFlNWE5NmM1MjVhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Ytyl0pok4HctdBUfKC3iZutH3nUQ9fjxO-OiuDNmow0) ![Screenshot (634)](https://private-user-images.githubusercontent.com/106414173/329840569-d84d4c59-6825-47ad-a486-2c026aa8bfb7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1MTU2MTcsIm5iZiI6MTcxNTUxNTMxNywicGF0aCI6Ii8xMDY0MTQxNzMvMzI5ODQwNTY5LWQ4NGQ0YzU5LTY4MjUtNDdhZC1hNDg2LTJjMDI2YWE4YmZiNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxMlQxMjAxNTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xMjRjMGUyZThiOTc5NmE1ZGM5MDMwMDUwODc0ZTgxYjllODI0MWY5ZTcwZWI0OWMyZmZmNzExMmVlZjQ5ODUzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.tbb-MRsFAmElPuuEAMzQpP3_azxkkvNfcSHKgbILJ8k) ![Screenshot (635)](https://private-user-images.githubusercontent.com/106414173/329840572-868cdac4-0b09-4418-a650-7865cdd9c44a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU1MTU2MTcsIm5iZiI6MTcxNTUxNTMxNywicGF0aCI6Ii8xMDY0MTQxNzMvMzI5ODQwNTcyLTg2OGNkYWM0LTBiMDktNDQxOC1hNjUwLTc4NjVjZGQ5YzQ0YS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxMlQxMjAxNTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jNWI3YTJmZTZhMWVlODM2NTNmZThhOWJkZGFmN2QwNDI2NWM5YWI1OWJjNmU3NzBiMTM3YTM1ODQ3OTNhYWZkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.3F0Did72LOibx-7UA9qNdryd-J_60nVd82Mtc2k-TkQ)

Yes Sir, I will start working on it and create issues.Thank you Sir.

**Comment by RakshitaJain2126** [comment]
Yes @govind-kumarr Sir, I will start working on it too and create issues. Thank you.

**Comment by palayushi293** [comment]
we will together work upon it

**Comment by RakshitaJain2126** [comment]
Yes @palayushi293.It would be great. Drop your LinkedIn Id.I will dm you and we will discuss.

**Comment by palayushi293** [comment]
https://www.linkedin.com/in/ayushi-pal-758634257/

**Comment by govind-kumarr** [comment]
Hi mates any update

**Comment by callmeVAMP** [comment]
My end sem exams are till 21st may. So can I start after that?

**Comment by govind-kumarr** [comment]
@callmeVAMP  No problem and All the best for exams.

**Comment by RakshitaJain2126** [comment]
@govind-kumarr I am Setting up the backend.It's first time so it is taking longer time for connecting MongoDB and ChromaDB.

**Comment by govind-kumarr** [comment]
@RakshitaJain2126  I can understand.  You can use a free MongoDB Atlas account but currently, there is no cloud version for Chroma so you will need to set it locally. You can check the readme file inside the backend dir. It's very easy with docker you just need to run a few commands. Let me know if you need any help.

**Comment by tejaswini4996** [comment]
Shall I be the part of this issue?
